cypress/e2e/utils/room-utils.ts

```diff
@@ -0,0 +1,51 @@
+import Chainable = Cypress.Chainable;
+
+export default class RoomUtils {
```

Comment:
alternatively, you could use `cy.createRoom` (for better independence between tests) :
https://github.com/tchapgouv/tchap-web-v4/blob/ac8216affe8a6303c8014a4696d48f124cd61087/cypress/support/client.ts#L40-L41

But if it works, fine, let's move forward.